src/typed-method-types/workflows/triggers/scheduled.ts

```diff
@@ -42,6 +44,18 @@ type BaseFrequencyType = {
   on_week_num?: 1 | 2 | 3 | 4 | -1;
 };
 
+type OnceFrequencyType = {
+  /** @description How often the trigger will activate */
+  type: typeof SCHEDULE_FREQUENCY.Once;
+};
+
+type HourlyFrequencyType =
+  & {
+    /** @description How often the trigger will activate */
+    type: typeof SCHEDULE_FREQUENCY.Hourly;
+  }
+  & Pick<BaseFrequencyType, "repeats_every">;
```

Comment:
As asked in this [PR](https://github.com/slackapi/deno-slack-api/pull/49#discussion_r1017317835), there aren't any limitation for repeats_every in API. Similarly, for monthly frequency too we aren't restricting it to just 12 months only. User can set it to repeat after every 1 hr or 12hrs or 50hrs, etc... Same is with existing `Monthly` frequency too where they can select to repeat it after n months.